src/client.rs

```diff
@@ -414,6 +414,14 @@ impl<T: Read + Write> Client<T> {
         }
     }
 
+    /// Yield the underlying connection for this Client.
+    /// This consumes `self` since the Client is not much use without
```

Comment:
```suggestion
    /// Yield the underlying connection for this Client.
    ///
    /// This consumes `self` since the Client is not much use without
```